src/System.Text.Json/ref/Configurations.props

```diff
@@ -4,6 +4,9 @@
     <BuildConfigurations>
       netcoreapp;
       uap;
+      <!-- Only included for testing and validating that the sources are netstandard compatible. -->
+      <!-- This is meant to help with producing a source package and not to ship a netstandard compatible binary. -->
+      netstandard;
```

Comment:
Did you try just adding netstandard to the src project? Or did that fail and required this as well?

Reply:
> did that fail and required this as well?

It's necessary.

```text
C:\Users\ahkha\.nuget\packages\microsoft.dotnet.build.tasks.configuration\1.0.0-beta.19064.4\build\Microsoft.DotNet.Build.Tasks.Configuration.targets(166,5): error : Could not find a configuration for ProjectReference 'E:\GitHub\Fork\corefx\src\/System.Text.Json/ref/System.Text.Json.csproj' from configurations 'netcoreapp;uap' when building 'System.Text.Json' for configuration 'netstandard' . [E:\GitHub\Fork\corefx\src\System.Text.Json\src\System.Text.Json.csproj]
```